analyzer/tests/functional/analyze_and_parse/test_files/filename_collision/a/f.cpp

```diff
@@ -0,0 +1,4 @@
+void f_a() {
+    int* pa;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'pa' is not assigned a value.](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4623413)